cinema/views.py

```diff
@@ -111,8 +107,11 @@ def get_queryset(self):
 
         date = self.request.query_params.get("date")
         if date:
+            year, month, day = date.split("-")
```

Comment:
these are strings again, it's better to get datetime here (`datetime.strptime` method) and pass it to the queryset.filter